recipes/seadex-essentials/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.8)
```

This can be bumped to 3.15 now.

```suggestion
cmake_minimum_required(VERSION 3.15)
```